CHANGELOG.md

```diff
@@ -1,11 +1,22 @@
-#Changelog
+# Changelog
 All notable changes to this project will be documented in this file.
-s
+
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+
+## [Unreleased]
+### Changed
+- Contact information and bangalore office address have been updated in the contact page.
+
+### Removed
+- Infratab.com link from footer has been removed.
+
+### Fixed
+- Contact page title typo has been fixed.
```

Comment:
- The typo in the Contact page's title is fixed

Reply:
@moosicalbovine I have updated the changes in the changelog as per the comment.